src/components/CloseButton/CloseButton.driver.ts

```diff
@@ -0,0 +1,36 @@
+import { StylableDOMUtil } from 'stylable/test-utils';
+import { buttonDriverFactory as coreButtonDriverFactory } from 'wix-ui-core/dist/src/components/Button/Button.driver';
+import { DriverFactory, BaseDriver, ComponentFactory } from 'wix-ui-test-utils/driver-factory';
+import style from './CloseButton.st.css';
+import { Skin, Size } from './constants';
+
+// TODO: move this to core
+export interface CoreButtonDriver extends BaseDriver {
```

Comment:
I don't see why to wait with it.  In any case, add documentation for the funcs and remember this is a component that should use the core Button so the methods should have the same name to wix-style-react driver as much as it can.

Reply:
1) I created PR for core: https://github.com/wix/wix-ui/pull/466/files
2) I don't think we need to conform to WSR API in the core (if we have a good reason not too. The Backoffice can be an adapter  for legacy API's, or even in WSR we can have an adapter).

Reply:
fixed